website/docs/cluster-management/advanced-cluster-management-topics/howto-inject-credentials-into-template.mdx

```diff
@@ -1,23 +1,15 @@
 ---
-title: CAPI Provider Identities
+title: How to Inject Credentials Into Your Template
 hide_title: true
 ---
 
 import TierLabel from "../_components/TierLabel";
 
-# CAPI Provider Identities <TierLabel tiers="Enterprise" />
+# How to Inject Credentials Into Your Template <TierLabel tiers="Enterprise" />
 
-## Multi-tenancy
+Weave GitOps _templates_ describe the properties of your cluster—how many nodes, what version of Kubernetes, etc. The _identity_ refers to which account will be used to create the cluster. When you render a template, you need credentials. We populate the template for you. Note that this is necessary for every user.
```

Comment:
```suggestion
Weave GitOps _templates_ describe the properties of your cluster—how many nodes, what version of Kubernetes, etc. The _identity_ refers to which account will be used to create the cluster. When you render a template, you may want to set the credentials to be used for this cluster, for example, if the cost is allocated to a specific team.

The rendered resource can be automatically configured with the selected credentials.

Credentials are injected into the following resources:
* AWSCluster, AWSManagedControlPlane
* AzureCluster, AzureManagedCluster
* VSphereCluster

If no credentials are selected, no changes will be applied, and the credentials used by your CAPI controller will be used as the default.
```

Reply:
Nope, somehow they didn't appear for me until now. Committing...